.github/workflows/test.yml

```diff
@@ -84,12 +84,22 @@ jobs:
             django: 'Django>=3.2,<3.3'
             experimental: false
             parallel: '--parallel'
+          - python: '3.8'
+            django: 'Django>=3.2,<3.3'
+            experimental: false
+            parallel: '--parallel'
+            postgres: 'postgres:15'
+          - python: '3.11'
+            django: 'Django>=4.2,<4.3'
+            notz: notz
+            experimental: false
+            parallel: '--parallel'
```

Comment:
Is it possible to *not* add more to the matrix? It seems wasteful of resources, especially if we could probably achieve the desired effect by specifying each of the existing matrix with a different version, e.g. 12, 13, 14, 15.

It's very unlikely that a newer version of postgres causes issues, but in the event that it does, I think it's enough to isolate the cause even if we combine it with our existing setups (i.e. not necessarily have a 'control' variable).

Reply:
I completely agree this isn't an ideal way to do it, I just copied the existing spirit of the tests.

In an ideal world, the only variable in the matrix for the postgres-specific tests would be the version of postgres, but that's not how the tests are currently setup. If we want to go that route (which we probably should), I'm happy to amend the PR.

Reply:
> In an ideal world, the only variable in the matrix for the postgres-specific tests would be the version of postgres, but that's not how the tests are currently setup.

Yeah but you'll need separate matrices to test the different values of the other variables as well, e.g. Python version, `notz`, `emailuser`, and Django version. That would result in spinning up many runners (unless we use tox so we can have part of the matrix defined inside the runner) with little gain, since we can combine the different scenarios together to get a subset that is enough to cover most cases (which I think is what the current matrix tries to achieve).

We currently have 5 matrix includes for the postgres job, that can be changed to use something like 12, 12, 13, 14, 15. Then we can update the description in:

https://github.com/wagtail/wagtail/blob/1dbf2ec2dfe81017543c5b1eee676ec1c83e5a3f/.github/workflows/test.yml#L23-L30

to something like

```yml
# Current configuration:
# - django 3.2, python 3.8, postgres:12, parallel
# - django 3.2, python 3.8, mysql
# - django 4.1, python 3.9, sqlite
# - django 4.2, python 3.10, mysql, parallel
# - django 4.1, python 3.10, postgres:12, parallel, USE_EMAIL_USER_MODEL=yes
# - django 4.2, python 3.11, postgres:13, parallel, DISABLE_TIMEZONE=yes
# - django stable/4.2.x, python 3.10, postgres:14 (allow failures)
# - django main, python 3.10, postgres:15, parallel (allow failures)
```